Header header
geometry_msgs/Vector3 position
string obj_class
float32 score
